Shared/analysis/GeoElementViewshed360.cpp

```diff
@@ -62,7 +64,7 @@ constexpr double c_defaultOffsetZ = 5.0;
 GeoElementViewshed360::GeoElementViewshed360(GeoElement* geoElement, AnalysisOverlay* analysisOverlay,
                                              const QString& headingAttribute, const QString& pitchAttribute, QObject* parent) :
   Viewshed360(new GeoElementViewshed(geoElement, c_defaultHorizontalAngle, c_defaultVerticalAngle, c_defaultMinDistance, c_defaultMaxDistance, 0.0, 0.0, parent), analysisOverlay, parent),
-  m_geoElement(geoElement),
+  m_geoElementSignaler(new GeoElementSignaler(geoElement, GeoElementUtils::toQObject(geoElement))),
```

Comment:
This is a self-destructing helper (m_geoElementSignaler is a `QPointer`). It should delete/invalidate itself whenever the GeoElement goes away.